### PR TITLE
Update Match Header

### PR DESCRIPTION
**Pull request by bguzryanto**
Match header work in progress.

Current header improvement
image

### CODE REVIEW

File: components/Match/Match.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.Header{
+	text-align: center;
+	display: block;
+	text-transform: uppercase;
+	margin-top: 0;
+    margin-bottom: 30px;
+}
+
+.DireWin {
```
Comment (howardchung) [inline components/Match/Match.css]: I think we should just have a predefined dire/radiant color in palette.css and use it for all Radiant/Dire and win/loss context classes

File: components/Match/MatchHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from "classnames";
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: single quote

File: components/Match/MatchHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import RaisedButton from 'material-ui/RaisedButton';
+import { transformations } from './../../utility';
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: I don't think the leading './' is necessary

File: components/Match/MatchHeader.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import matchStyle from './Match.css';
+console.log(constants)
+
+export default ({ match }) => (
+  <div>
+    {match.radiant_win ? <RadiantWinText /> : <DireWinText />}
+    <table className={matchStyle.Table}>
+      <thead>
+        <tr>
+          <th className={matchStyle.TableCell}>MatchID</th>
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: ID (shorter)

File: components/Match/MatchHeader.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+          <th className={matchStyle.TableCell}>Mode</th>
+          <th className={matchStyle.TableCell}>Bracket</th>
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: Skill

File: components/Match/MatchHeader.jsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+          <th className={matchStyle.TableCell}>Region</th>
+          <th className={matchStyle.TableCell}>Duration</th>
+          <th className={matchStyle.TableCell}>Ended</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td className={matchStyle.TableCell}>{match.match_id}</td>
+          <td className={matchStyle.TableCell}>{transformations.game_mode(0, 0, match.game_mode)}</td>
```
Comment (howardchung) [inline components/Match/MatchHeader.jsx]: what's with the 0,0?

File: utility/index.js [hunk 7]
```diff
@@ -81,6 +81,12 @@ export const transformations = {
   start_time: (row, col, field) => (Number(field) ? moment(field, 'X').fromNow() : 'never'),
   last_played: (row, col, field) => (Number(field) ? moment(field, 'X').fromNow() : 'never'),
   duration: (row, col, field) => formatSeconds(field),
+  region: (row, col, field) => (Object.keys(constants.regions).find((region, index) => {
+      if (constants.regions[region].region == field) {
```
Comment (howardchung) [inline utility/index.js]: ===

**Comment by howardchung** [comment]
Reopen against master branch

File: components/Match/MatchHeader.jsx (hunk 2 above)
Comment (bguzryanto) [inline components/Match/MatchHeader.jsx]: yea, Its just some PoC

File: components/Match/MatchHeader.jsx (hunk 6 above)
Comment (bguzryanto) [inline components/Match/MatchHeader.jsx]: Ok I will change it

**Pull request closed without merging**